src/k8s-extension/setup.py

```diff
@@ -33,7 +33,7 @@
 # TODO: Add any additional SDK dependencies here
 DEPENDENCIES = []
 
-VERSION = "1.4.2"
+VERSION = "1.4.3"
```

Comment:
```suggestion
VERSION = "2.0.0"
```
Since this PR includes breaking changes, we usually recommend upgrading the major version